src/player.ts

```diff
@@ -146,7 +146,6 @@ class Player extends LitElement {
         position: relative;
         background: var(--sonos-int-background-color);
         border-radius: var(--sonos-int-border-radius);
-        border: 8px solid var(--sonos-int-background-color);
```

Comment:
A border here is probably nice when having a background image in your lovelace theme. Could we make it themable?

Reply:
You're right! I didn't think of that. It could be like this `border: var(--sonos-player-border) solid var(--sonos-int-background-color);`